src/routes/volume/detail/index.js

```diff
@@ -718,6 +750,7 @@ function VolumeDetail({ snapshotModal, dispatch, backup, engineimage, eventlog,
             <EventList {...eventListProps} />
           </Col>
         </Row>
+        {updateBackupTargetModalVisible && <UpdateBackupTargetModal {...updateBackupTargetModalProps} />}
```

Comment:
[nit] The modal already handles its visibility, so `updateBackupTargetModalVisible &&` is unnecessary here.